views/includes/inputs/fields/is-still-a-managing-officer-input.html

```diff
@@ -0,0 +1,46 @@
+{% if managingOfficerPronoun == "they" %}
+  {% set dateText = "Date they stopped being a managing officer" %}
+  {% set stillMoText = "Are they still a managing officer for the overseas entity?" %}
+{% else %}
+  {% set dateText = "Date it stopped being a managing officer" %}
+  {% set stillMoText = "Is it still a managing officer for the overseas entity?" %}
```

Comment:
Can't see a screenshot on ticket. Where you get the content?

Reply:
I got them from the prototype links on the ticket but it seems like the password on there no longer works :(

Reply:
Okay no bother, I tried to check the prototype and wasn't able to get in but happy with that answer :D